stdlib/std.jsonnet

```diff
@@ -1703,6 +1703,16 @@ limitations under the License.
 
   sum(arr):: std.foldl(function(a, b) a + b, arr, 0),
 
+  minArray(arr):: 
```

Comment:
Wouldn't it be more useful with keyF argument, as in std.sort/std.set/std.uniq?

Reply:
Correct, I have made the required changes. Thanks!